exercises/pov/Example.cs

```diff
@@ -0,0 +1,125 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Linq;
+
+public class Graph<T> : IEquatable<Graph<T>>
+{
+    public Graph(T value, IEnumerable<Graph<T>> children)
+    {
+        Value = value;
+        Children = children;
+    }
+
+    public T Value { get; }
+    public IEnumerable<Graph<T>> Children { get; }
+
+    public bool Equals(Graph<T> other) =>
+        Value.Equals(other.Value) && Children.SequenceEqual(other.Children);
+}
+
+public class Crumb<T>
+{
+    public Crumb(T value, IEnumerable<Graph<T>> left, IEnumerable<Graph<T>> right)
+    {
+        Value = value;
+        Left = left;
+        Right = right;
+    }
+
+    public T Value { get; }
+    public IEnumerable<Graph<T>> Left { get; }
+    public IEnumerable<Graph<T>> Right { get; }
+}
+
+public class Zipper<T>
+{
+    public Zipper(Graph<T> focus, IEnumerable<Crumb<T>> crumbs)
+    {
+        Focus = focus;
+        Crumbs = crumbs;
+    }
+
+    public Graph<T> Focus { get; }
+    public IEnumerable<Crumb<T>> Crumbs { get; }
+}
+
+public static class Pov
+{
+    public static Graph<T> CreateGraph<T>(T value, IEnumerable<Graph<T>> children) 
+        => new Graph<T>(value, children);
+
+    public static Graph<T> FromPOV<T>(T value, Graph<T> graph) where T : IComparable 
+        => ChangeParent(FindNode(value, GraphToZipper(graph)));
+
+    public static IEnumerable<T> TracePathBetween<T>(T value1, T value2, Graph<T> graph) where T : IComparable
+        => ZipperToPath(FindNode(value2, GraphToZipper(FromPOV(value1, graph))));
+
+    private static Zipper<T> GraphToZipper<T>(Graph<T> graph)
+    {
+        if (graph == null)
+            return null;
+
+        return new Zipper<T>(graph, Enumerable.Empty<Crumb<T>>());
+    }
+
+    private static IEnumerable<T> ZipperToPath<T>(Zipper<T> zipper)
+    {
+        if (zipper == null)
+            return null;
+
+        return zipper.Crumbs.Select(c => c.Value).Reverse().Concat(new[] { zipper.Focus.Value });
```

Comment:
Do you have any specific examples in mind where you feel `?.` obscures the code? I'm curious because I've found that using it has simplified a lot of "noise" in the code by doing null checks all over the place.